service/src/test/java/bio/terra/landingzone/library/landingzones/TestArmResourcesFactory.java

```diff
@@ -83,7 +83,7 @@ public static ResourceGroup createTestResourceGroup(AzureResourceManager azureRe
     return azureResourceManager
         .resourceGroups()
         .define("test-" + resourceGroupId)
-        .withRegion(Region.US_EAST2)
+        .withRegion(Region.US_EAST)
```

Comment:
I think we bumped some quota for tests. @sergiygetlin is it OK to have changed the region here?